src/kvazaar.h

```diff
@@ -34,6 +34,10 @@
 extern "C" {
 #endif
 
+#if defined(KVZ_STATIC_LIB) && defined(PIC)
+  #undef KVZ_STATIC_LIB
```

Comment:
Apparently gcc, clang and mingw32 require `KVZ_PUBLIC` to be defined as `__attribute__ ((visibility ("default")))` for dll builds (not required for static builds but it doesn't seem to do any harm either):
``` C
#if defined(__GNUC__)
  // Using GCC.
  #define KVZ_PUBLIC __attribute__ ((visibility ("default")))
#elif defined(_WIN32)
  // Using windows and not GCC.
  #if defined(KVZ_STATIC_LIB)
    // Using or building kvazaar as a static library on windows.
    #define KVZ_PUBLIC
  #elif defined(KVZ_DLL_EXPORTS) || defined(DLL_EXPORT)
    // Building kvazaar on windows.
    #define KVZ_PUBLIC __declspec(dllexport)
  #else
    // Using kvazaar as a DLL on windows.
    #define KVZ_PUBLIC __declspec(dllimport)
  #endif
#else
  #define KVZ_PUBLIC
#endif
```
This way `KVZ_DLL_EXPORTS` and `KVZ_STATIC_LIB` don't need to be defined in configure.ac or Makefile.am. But I'm not sure how handle non-gcc compatible compilers invoked through configure (though that might be relatively unlikely?).